### ISSUE TITLE
Notification on booting Windows if streamer is already live

### ISSUE THREAD

**Issue by gorro123** [mmozeiko/TwitchNotify#7]
Is there a chance to implement the feature, so you get a notification on starting up Windows, if the streamer is already live?

**Comment by sir-pinecone** [comment]
I'm not sure how Martins wants to handle this, but this change I made in my fork implements your request. I was actually just thinking of adding support when I noticed your issue. Note that this will also show a notification when adding a user to the ini and they happen to be live.

https://github.com/sir-pinecone/TwitchNotify/commit/3f52585b9a

**Comment by mmozeiko** [comment]
I'm fine with such feature. But it should be optional feature, meaning having a checkbox to enable "Notify on startup" or something like that.

**Comment by sir-pinecone** [comment]
Yeah I can see it being annoying if you follow a lot people and they're live. I might add the checkbox some time soon, but otherwise in the meantime @gorro123 you can use the commit I linked to compile a new build for yourself.

**Comment by mmozeiko** [comment]
This is now available in latest build.
